src/chem/selectors.test.js

```diff
@@ -0,0 +1,1066 @@
+import chai, { expect } from 'chai';
+import dirtyChai from 'dirty-chai';
+import proxyquire from 'proxyquire';
+import selectors from './selectors';
+import Atom from './Atom';
+import ResidueType from './ResidueType';
+import utils from '../utils';
+
+chai.use(dirtyChai);
+
+describe('selectors', () => {
```

Comment:
The file is overwhelmingly big and hard to understand. So many helper variables. So many test cases. I think we can merge some tests together. I'm afraid I cannot provide useful comments right now, I'll look at the file and see what I can suggest/do.